website/content/partials/cli/token/flags/format.mdx

```diff
@@ -0,0 +1,19 @@
+<a id="global-format" />
```

Comment:
We should use the global `-format` flag and update the default for that flag instead of creating a new partial